src/MeetingsSDKAdapter.js

```diff
@@ -750,7 +750,7 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
         receiveShare: true,
       });
 
-      // The rest of the cleanup is done in the handling of the EVENT_LOCAL_SHARE_STOP event emitted by sdkMeeting.updateShare
```

Comment:
let's add a listener (before the `await` above) for the `EVENT_LOCAL_SHARE_STOP` event,

```js
sdkMeeting.once(EVENT_LOCAL_SHARE_STOP, () => {
  this.stopStream(this.meetings[ID].localShare);
  this.meetings[ID].localShare = null;
});